src/index.tsx

```diff
@@ -2,12 +2,12 @@ import arrayMove from 'array-move'
 import React, { HTMLAttributes } from 'react'
 
 import { findItemIndexAtPosition } from './helpers'
-import { useDrag } from './hooks'
+import { useDrag, usePlaceholder } from './hooks'
 import { Point } from './types'
 
 const DEFAULT_CONTAINER_TAG = 'div'
 
-type Props<TTag extends keyof JSX.IntrinsicElements> = HTMLAttributes<TTag> & {
+type Props<TTag extends keyof JSX.IntrinsicElements> = Omit<HTMLAttributes<TTag>, 'placeholder'> & {
```

Comment:
We need to omit placeholder prop because it conflicts with the native HTMLAttribute

Reply:
What do you think of calling this prop/feature: `dropTarget`?

Reply:
Yeah that might make more sense. I updated all naming in code, including story and README as well.